src/Minesweeper.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/50989355/135903850-bbf44242-c99c-48b9-8bea-38787f07134d.png)
# When I try to run this code with `npm start` I get this error, please look at that.

Reply:
🚩